moped-database/views/component_arcgis_online_view.sql

```diff
@@ -225,4 +229,5 @@ LEFT JOIN moped_components mc ON mpc.component_id = mc.component_id
 LEFT JOIN related_projects rp ON mpc.project_id = rp.project_id
 LEFT JOIN latest_public_meeting_date lpmd ON mpc.project_id = lpmd.project_id
 LEFT JOIN earliest_active_or_construction_phase_date eaocpd ON mpc.project_id = eaocpd.project_id
+LEFT JOIN LATERAL (SELECT timezone('US/Central'::text, get_project_development_status_date(lpmd.latest::timestamp with time zone, eaocpd.earliest, coalesce(mpc.completion_date, plv.substantial_completion_date), plv.substantial_completion_date_estimated, coalesce(mph.phase_name_simple, current_phase.phase_name_simple))) AS result) project_development_status_date ON true
```

Comment:
Noting to myself that the SQLFluff GH action made a change here:
`LEFT JOIN LATERAL (SELECT get_project_development_status_date(lpmd.latest::timestamp with time zone, eaocpd.earliest, coalesce(mpc.completion_date, plv.substantial_completion_date), plv.substantial_completion_date_estimated, coalesce(mph.phase_name_simple, current_phase.phase_name_simple)) AT TIME ZONE 'US/Central' AS result) AS project_development_status_date ON true -- noqa: ST05`

Reply:
I tested the output of the pre- and post-linted version of this view, and I'd like for the version in the migration to match up with what shows up in our reference file so I decided to update the migration with the post-linted version.

Reply:
This was actually from Postgres since this GH action uses SQLFluff for linting, creates the view in the DB, and then reads what Postgres created and store out of the DB to create this output.

Reply:
side note is it necessary for `project_development_status_date` to be a timestamp bc it seems like there is no customizing the time for that column its just selecting a date and time is always gonna be `00:00:00`?

Reply:
@roseeichelmann , Amenity asked the same question, and the answer is timezones!
https://austininnovation.slack.com/archives/CNUEPKLB1/p1725395804733349

Reply:
i will add that when in Moped v1.0 a lot of these columns were actually the postres `date` column type, which holds just a naive date string like `2020-01-01`. and we had timezone because all of those dates were stored from local time!

Reply:
okay i was wondering if the timezone bugs were the reason, thanks guys! time is so :face_with_spiral_eyes: